### ISSUE TITLE
Use note names

### ISSUE THREAD

**Issue by isaac-dunn** [cambridge-alpha-team/unvisual-frontend#9]
If you want a particular note, it's definitely not obvious which number you should use. Perhaps we could play the notes as you're selecting which number you want, or we could use note names?

**Comment by tjvr** [comment]
Note names would be great. Some of our users were musical, so they'd have liked that.

> Perhaps we could play the notes as you're selecting which number you want

I don't think we should do that.

In #10, I talked about immediately updating values without confirmation. That will make it easier to get feedback about which note you played—simply change the value up or down, and press `↩`.

**Comment by isaac-dunn** [comment]
And press enter, presumably.

Are we going to use note names only, then, or have a way of choosing the representation that you want?

**Comment by LauraneS** [comment]
Could we have more names at the top of the list and then int values if they don't want to use note names?

**Comment by isaac-dunn** [comment]
I'm not sure how that would work - you'd have to select which octave you wanted the note names to be played in, and I assumed that we'd manage this by starting at the bottom of one octave when you reached the top of the octave before (F4, G4, A5, B5, etc.), which implies it'd be a very long list before you reached the end of it. If we could have another way of selecting octave then your suggestion could work, though?

**Comment by tjvr** [comment]
> you'd have to select which octave you wanted the note names to be played in

What?

**Comment by isaac-dunn** [comment]
I thought Lulu was suggesting a list like this:

```
A, B, C, D, E, F, G, 40, 41, 42, ...
```

(which on reflection wouldn't be great anyway because you'd have to start the numbers at the min or max value), and I was saying you'd need a way to indicate which octave the note was if you chose the letter name. I was pointing out that a list like

```
 A1, B1, C1, D1, E1, F1, G1, A2, B2, ..., G8, 40, 41, 42, ...
```

definitely wouldn't work as a way of selecting the octave.

**Comment by isaac-dunn** [comment]
Maybe I got the wrong end of the stick about what you mean, @LauraneS ?

**Comment by LauraneS** [comment]
No that is what I was thinking of, but I forgot to factor in the octave choice

**Comment by tjvr** [comment]
Are you suggesting adding an extra level of hierarchy, and having the user select the octave number and note name separately? As a musician, I don't think users will like that—musical or otherwise.

I think keeping the same range as we have currently, but replacing the note numbers with names (like `A1`), would be excellent.

**Comment by isaac-dunn** [comment]
I agree that this would be preferable to numbers, which are basically meaningless.
